po/fr.po

```diff
@@ -1,5 +1,723 @@
+#: src/Services/ActionManager.vala:243 src/Services/ActionManager.vala:262
+#, c-format
+msgid "Unable to open file at '%s'"
+msgstr "Impossible d'ouvrir le fichier « %s »"
+
+#: src/Layouts/MainCanvas.vala:46
+msgid "Button was pressed!"
+msgstr "Un bouton a été appuyé !"
+
+#: src/Layouts/MainCanvas.vala:206
+msgid "Export completed!"
+msgstr "Exportation complète !"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:48
+msgid "Distribute Horizontally"
+msgstr "Distribuer horizontalement"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:50
+msgid "Distribute Vertically"
+msgstr "Distribuer verticalement"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:52
+msgid "Align Left"
+msgstr "Aligner à gauche"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:53
+msgid "Align Center"
+msgstr "Aligner au centre"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:54
+msgid "Align Right"
+msgstr "Aligner à droite"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:56
+msgid "Align Top"
+msgstr "Aligner en haut"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:57
+msgid "Align Middle"
+msgstr "Aligner au milieu"
+
+#: src/Layouts/Partials/AlignItemsPanel.vala:58
+msgid "Align Bottom"
+msgstr "Aligner en bas"
+
+#: src/Layouts/Partials/Artboard.vala:144
+#: src/Layouts/Partials/Artboard.vala:487 src/Layouts/Partials/Layer.vala:139
+#: src/Layouts/Partials/Layer.vala:687
+msgid "Lock Layer"
+msgstr "Verrouiller le calque"
+
+#: src/Layouts/Partials/Artboard.vala:155
+#: src/Layouts/Partials/Artboard.vala:529 src/Layouts/Partials/Layer.vala:156
+#: src/Layouts/Partials/Layer.vala:729
+msgid "Hide Layer"
+msgstr "Cacher le calque"
+
+#: src/Layouts/Partials/Artboard.vala:487 src/Layouts/Partials/Layer.vala:687
+msgid "Unlock Layer"
+msgstr "Dévérouiller le calque"
+
+#: src/Layouts/Partials/Artboard.vala:529 src/Layouts/Partials/Layer.vala:729
+msgid "Show Layer"
+msgstr "Afficher le calque"
+
+#: src/Layouts/Partials/BorderItem.vala:121
+msgid "Choose border color"
+msgstr "Choisir la couleur de la bordure"
+
+#: src/Layouts/Partials/BorderItem.vala:176
+#: src/Layouts/Partials/FillItem.vala:178
+msgid "Pick color"
+msgstr "Choisir une couleur"
+
+#: src/Layouts/Partials/BorderItem.vala:191
+msgid "Remove border"
+msgstr "Supprimer la bordure"
+
+#: src/Layouts/Partials/BorderItem.vala:275
+msgid "Show border"
+msgstr "Afficher la bordure"
+
+#: src/Layouts/Partials/BorderItem.vala:279
+msgid "Hide border"
+msgstr "Masquer la bordure"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:86
+msgid "Style"
+msgstr "Style"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:103
+msgid "Border Radius"
+msgstr "Arrondi"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:137
+msgid "Border radius options"
+msgstr "Options de l'arrondi"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:199
+msgid "Autoscale Corners"
+msgstr "Mise à l'échelle automatique des coins"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:208
+msgid "Uniform Corners"
+msgstr "Uniformiser les bordures"
```

Comment:
I need context to valid this one (poke @albfan)

Reply:
Context:
![Peek 12-10-2020 21-01](https://user-images.githubusercontent.com/22885898/95781493-34966580-0cce-11eb-9a80-6bb4373d9a1e.gif)
(the management of rounding for a single corner does not seem to work)

Reply:
I would say
> Coins uniformes

in this case because it's a checkbox, not an action